dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -128,6 +128,10 @@ error.kernelDefFile=CWWKE0023E: The specified kernel definition ({0}) does not e
 error.kernelDefFile.explanation=The specified kernel definition could not be found.
 error.kernelDefFile.useraction=Ensure that the kernel definition resource exists and can be read by the current user.
 
+error.badOptionValue=CWWKE0024E: The value ({0}) for the option ({1}) is not valid.
```

Comment:
```suggestion
error.badOptionValue=CWWKE0024E: The ({0}) value for the ({1}) option  is not valid.
```

Reply:
do the variables need ({double brackets}) ? Unless we expect readability to be an issue, I'd remove the outter (parentheses). If the potential values are especially long or difficult to read, ok as is.

Reply:
The {0} is replaced by a value.   So in the displayed message, one would only see ( substituted text ).   I think it is helpful to separate the command line options and values which must be taken verbatim from the rest of the text which is translated.  For example: The ( 5mm ) value for the (--timeout) option is not valid.